src/alvarium/contracts/annotation.py

```diff
@@ -12,6 +12,7 @@ class AnnotationType(Enum):
     PKI = "pki"
     TLS = "tls"
     SOURCE = "src"
+    HTTPPKI = "httppki"
```

Comment:
```suggestion
    HTTPPKI = "pki-http"
```
I think we should change this to conform with the [Go](https://github.com/project-alvarium/alvarium-sdk-go/blob/f54efd547ae5bfe6530b7cf8c53047747718c33c/pkg/contracts/constants.go#L70) and Java SDKs.